src/App.tsx

```diff
@@ -27,8 +27,13 @@ export const AppBody = styled.div`
   justify-content: center !important;
 `
 
+const AppContainer = styled.div`
```

Comment:
We're you planning on doing something with this styled component or were just renaming the div to something more semantic?